msg/en_US/csql.msg

```diff
@@ -188,6 +191,13 @@ $ csql help messages
    ;HISTORYRead <history_num>   - read entry on the history number into command buffer.\n\
    ;TRAce [ON/OFF] [text/json]  - enable/disable sql auto trace.\n\
    ;SIngleline [ON|OFF]         - enable/disable single-line mode.\n\
+   ;Connect [user-name] [database-name]@[host-name]\n\
```

Comment:
;Connect username [dbname][@hostname]\n\

Reply:
Thank you. I will apply it.